### DISCUSSION TITLE
Image-quality of the UAvatar-component in Chrome vs. Firefox

### DISCUSSION THREAD

**Discussion by jakobmatzen** [nuxt/nuxt#26476] [Questions]
Hi, I'm currently building an application which gets accessed by different users. Each user has a profile-picture, which gets shown through a UAvatar-component. I noticed that the quality of the image is pretty bad and I was trying out different solutions to fix that. At some point I was using Firefox instead of Chrome and suddenly the profile-pictures were shown in perfect quality. Is there a difference between those two browsers in how they handle the rendering of the images, which may affect the quality of the UAvatar-component? I access the images through a web-url, but I also tried a local file and it was the same issue. I can also provide code if needed. Thanks in advance!

Firefox: ![firefox](https://github.com/nuxt/nuxt/assets/121267958/7ffc1d1c-0379-4ba2-9f9d-359ade91c50d)
Firefox with local image: ![local_firefox](https://github.com/nuxt/nuxt/assets/121267958/54483f1b-10d6-4f9e-9b30-ea10a1aa5f85)

Chrome: ![chrome](https://github.com/nuxt/nuxt/assets/121267958/3dea0f31-f727-4904-a2c5-034b855a5f6b)
Chrome with local image: ![local_chrome](https://github.com/nuxt/nuxt/assets/121267958/e2ea5efc-9a23-4567-8aa5-b704c9fb26eb)

**Comment by MichaUFL** [comment]
Same problem here - any solution?
